libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/NSString+SFAdditions.h

```diff
@@ -15,8 +31,6 @@ typedef NS_ENUM(NSUInteger, SFEntityIdLength) {
     SFEntityIdLengthMax = SFEntityIdLength18
 };
 
-extern NSString * const SFUserAgentAppName;
```

Comment:
The implementation of this class hardcodes this to an internal product name. User agent shouldn't be coming from here anyway.